classes/news/NewsAPIManager.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+require __DIR__ . '/NewsPostDto.php';
+
+class NewsAPIManager {
```

Comment:
```suggestion
class NewsManager {
```
I realize I didn't present this consistently, but I don't think we should include `API` in these manager classes, it's overly specific. We want to use them for anything, not just APIs (as opposed to the presenter and response which specifically are only useful for constructing a response)

Reply:
Fixed in commit.